### PR TITLE
games-rpg/baldurs-gate-ee: new package

### PR DESCRIPTION
**Pull request by diogocp**
Baldur's Gate Enhanced Edition from [GOG.com](https://www.gog.com/game/baldurs_gate_enhanced_edition).

I get the following QA notice on install:
```
 * QA Notice: Found an absolute symlink in a library directory:
 *            opt/baldurs-gate-ee/lib/libjson.so.0 -> /usr/lib32/libjson-c.so
 *            It should be a relative symlink if in the same directory
 *            or a linker script if it crosses the /usr boundary.
```
Let me know if I need to handle this somehow. I looked into linker scripts and it didn't seem to make much sense in this context.

### CODE REVIEW

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+S="${WORKDIR}/data/noarch"
+
+pkg_nofetch() {
+	einfo "Please buy and download \"${SRC_URI}\" from"
+	einfo "https://www.gog.com/game/baldurs_gate_enhanced_edition"
+	einfo "and copy it to \"${DISTDIR}\""
+}
+
+src_unpack() {
+	unpack_zip "${DISTDIR}/${SRC_URI}"
```
Comment (mgorny) [inline games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild]: Why do you use unpacker? Doesn't the default zip unpacking function work for you?

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+	dodoc -r "game/Manuals/."
+	rm -r "game/Manuals" || die "rm failed"
+
+	insinto "${dir}"
+	doins -r "game/."
+	fperms +x "${dir}/BaldursGate"
+
+	dodir "${dir}/lib"
+	dosym "/usr/$(get_abi_LIBDIR x86)/libjson-c.so" "${dir}/lib/libjson.so.0"
```
Comment (mgorny) [inline games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild]: 1. I'd say it'd be safer to set `local ABI=x86` on top of `src_install`.
2. Then you just use regular `get_libdir` and don't have to inherit multilib.
3. To kill the QA warning, just put the relative path there, i.e. `../../usr/`... (if I count slashes right).

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild (hunk 1 above)
Comment (mgorny) [inline games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild]: Oh right, it's fine then.

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild (hunk 2 above)
Comment (diogocp) [inline games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.3.67.3.ebuild]: Done. You didn't count the slashes right :wink:

**Pull request closed without merging**